stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/ChatApi.kt

```diff
@@ -72,93 +73,41 @@ internal class ChatApi(
         channelId: String,
         file: File,
         callback: ProgressCallback
-    ) {
-        val body = ProgressRequestBody(
-            file,
-            callback
-        )
-        val part = createFormData("file", file.name, body)
-
-        retrofitCdnApi
-            .sendFile(
-                channelType,
-                channelId,
-                part,
-                apiKey,
-                userId,
-                connectionId
-            )
-            .call
-            .enqueue(RetroProgressCallback(callback))
-    }
+    ): Unit = fileUploader.sendFile(channelType, channelId, userId, connectionId, file, callback)
```

Comment:
We have _a lot_ of String parameters here, it would be a good idea to add names to the arguments being passed in, so that we clearly see that they're in the right position.